### PR TITLE
Fix ServicePrincipalNames description

### PR DESCRIPTION
**Pull request by X-Guardian**
The current `ServicePrincipalNames` description is completely wrong. Replaced with the text from the updated `Get-Help` description.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/addsadministration/set-aduser.md [hunk 1]
```diff
@@ -1312,10 +1312,39 @@ Accept wildcard characters: False
 ```
 
 ### -ServicePrincipalNames
-Use the **DateTime** syntax when you specify this parameter.
-Time is assumed to be local time unless otherwise specified.
-When a time value is not specified, the time is assumed to 12:00:00 AM local time.
-When a date is not specified, the date is assumed to be the current date.
+Specifies the service principal names for the account. This parameter sets the ServicePrincipalNames property of the account. The LDAP display name (ldapDisplayName) for this property is servicePrincipalName. This parameter uses the following syntax to add remove, replace or clear service principal name values.
```
Comment (JohanFreelancer9) [inline docset/windows/addsadministration/set-aduser.md]:
```suggestion
Specifies the service principal names for the account. This parameter sets the ServicePrincipalNames property of the account. The LDAP display name (ldapDisplayName) for this property is servicePrincipalName. This parameter uses the following syntax to add, remove, replace or clear service principal name values.
```

**Comment by e0i** [comment]
@X-Guardian Thank you for following up with copy editing suggestions.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by tseward** [comment]
Looks good.

**Comment by get-itips** [comment]
Looks ok.

**Comment by dariomws306** [comment]
I am ok with this one.

**Comment by e0i** [comment]
@AndreaBarr This PR that has passed review is ready to be merged.

**Pull request merged**